src/chart/parameter/ParameterSelectionChart.tsx

```diff
@@ -97,6 +100,7 @@ export const NeoParameterSelectionChart = (props: ChartProps) => {
         settings={props.settings}
```

Comment:
multiselect doens't make sense for dates

Reply:
fixed